### ISSUE TITLE
Handle non-UTF-8 strings 

### ISSUE THREAD

**Issue by fleimgruber** [AurelienLourot/uncommitted#10]
At some point in the output of `uncommitted -Luns` I get
```
Traceback (most recent call last):
  File "D:\Miniconda3_64\envs\py35\Scripts\uncommitted-script.py", line 11, in <module>
    load_entry_point('uncommitted', 'console_scripts', 'uncommitted')()
  File "d:\uncommitted\uncommitted\command.py", line 210, in main
    scan(repos, options.verbose)
  File "d:\uncommitted\uncommitted\command.py", line 148, in scan
    lines = get_status(directory, ignore_set)
  File "d:\uncommitted\uncommitted\command.py", line 122, in status_subversion
    for line in run(['svn', 'st', '-v'], cwd=path):
  File "d:\uncommitted\uncommitted\command.py", line 31, in run
    return output.decode().splitlines()
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe4 in position 4491: invalid continuation byte
```

I read up on this[1] and following that the most likely cause is a Latin1 encoded string. From what I know on this topic, one would need to know the encoding of the bytestring. In this case, the encodings are most likely Latin1, cp850 or cp1252 (Windows...). Thinking in terms of a solution, one could try a list of candidate encodings for the `.decode()` method, catch decoding exceptions until one succeeds or re-raise the above exception if no encoding worked. What do you think?

[1] http://stackoverflow.com/questions/5552555/unicodedecodeerror-invalid-continuation-byte#5552623

**Comment by AurelienLourot** [comment]
Thanks for your analysis @fleimgruber. This should now be fixed in 27f7ff05981fc5924ecf0766fee071f054467a25. Do you confirm?

As you can see, I've decided to just use `decode(errors='replace')`, which simply replaces unknown chars by question marks.

**Comment by fleimgruber** [comment]
Yep, I can confirm this works nicely. Thanks for the prompt fix.

Keep up the good work!
